main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -20,14 +20,29 @@
                 }
             };
 
+            var BingoCell = function () {
+                this.bingoNumber = null;
+                this.marked = "unmarked";
+                this.counted = false;
+            };
+
+            var TicketInfo = function () {
+                this.row1Marked = 0;
```

Comment:
These values should be calculated - the sum of the "togo" values in each square.